### PR TITLE
New package: supertux2-0.3.5a

### PR DESCRIPTION
**Pull request by beefcurtains**
One of #2097 :smile_cat:

### CODE REVIEW

File: srcpkgs/supertux2/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+depends="${sourcepkg}-data"
+short_desc="Side-scrolling platform game, milestone 2"
+maintainer="beefcurtains <beefcurtains@voidlinux.eu>"
+license="GPL-3"
+homepage="https://supertux.github.io/"
+distfiles="https://github.com/SuperTux/supertux/releases/download/v${version}/supertux-${version}.tar.bz2"
+checksum=71c34502b5879a4130fb111ac754967c3a632ef3017f30883a4d496d87bef8c7
+
+supertux2-data_package() {
+	pkg_install() {
```
Comment (xtraeme) [inline srcpkgs/supertux2/template]: noarch ?

**Comment by pullmoll** [comment]
:+1:

**Comment by beefcurtains** [comment]
I wonder why this happens during the build
```
xbps-uhelper: invalid option -- 'a'
xbps-uhelper: invalid option -- 'a'
usage: xbps-uhelper [options] [action] [args]

  Available actions:
    binpkgarch, binpkgver, cmpver, digest, fetch, getpkgdepname,
    getpkgname, getpkgrevision, getpkgversion, pkgmatch, version,
    real-version, arch, getsystemdir

  Action arguments:
    binpkgarch	<binpkg>
    binpkgver	<binpkg>
    cmpver		<instver> <reqver>
    digest		<file> <file1+N>
    fetch		<URL[>filename]> <URL1+N[>filename]>
    getpkgdepname	<string>
    getpkgdepversion	<string>
    getpkgname		<string>
    getpkgrevision	<string>
    getpkgversion	<string>
    pkgmatch		<pkg-version> <pkg-pattern>
    version		<pkgname>
    real-version	<pkgname>

  Options shared by all actions:
    -C		Path to xbps.conf file.
    -d		Debugging messages to stderr.
    -r		<rootdir>
    -V		Prints the xbps release version

  Examples:
    $ xbps-uhelper cmpver 'foo-1.0_1' 'foo-2.1_1'
    $ xbps-uhelper digest file ...
    $ xbps-uhelper fetch http://www.foo.org/file.blob ...
    $ xbps-uhelper getpkgdepname 'foo>=0'
    $ xbps-uhelper getpkgdepversion 'foo>=0'
    $ xbps-uhelper getpkgname foo-2.0_1
    $ xbps-uhelper getpkgrevision foo-2.0_1
    $ xbps-uhelper getpkgversion foo-2.0_1
    $ xbps-uhelper pkgmatch foo-1.0_1 'foo>=1.0'
    $ xbps-uhelper version pkgname
   [runtime] -data: found (xbps-query)
```

File: srcpkgs/supertux2/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+depends="${sourcepkg}-data"
+short_desc="Side-scrolling platform game, milestone 2"
+maintainer="beefcurtains <beefcurtains@voidlinux.eu>"
+license="GPL-3"
+homepage="https://supertux.github.io/"
+distfiles="https://github.com/SuperTux/supertux/releases/download/v${version}/supertux-${version}.tar.bz2"
+checksum=71c34502b5879a4130fb111ac754967c3a632ef3017f30883a4d496d87bef8c7
+
+supertux2-data_package() {
+	noarch=yes
```
Comment (xtraeme) [inline srcpkgs/supertux2/template]: This is missing `short_desc` too.

**Comment by xtraeme** [comment]
For some reason `$sourcepkg` in `depends` is not being expanded and xbps-uhelper detects it as an invalid pkgver.

**Pull request merged**